aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -119,11 +121,27 @@
             sleep 5s
             cd -
       path: /home/ubuntu/install-and-run-helics-scripts.sh
+    - content: |
+            #!/bin/bash
+            timeout=$1
+            echo "date,time,CPU usage,RAM used,RAM available"
+            while sleep $timeout
+            do
+                    timestamp_CPU=$(vmstat 1 3 -SM -a -w -t | python3 -c 'import sys; ll=sys.stdin.readlines()[-1].split(); print(ll[-2] + ", " + ll[-1] + ", " + str(100 - int(ll[-5])))')
```

Comment:
it looks like `vmstat -t` will give enough information and you have just one line to parse

Reply:
It won't - it will give stats from the start of the system. 
This is why I need to specify two numbers, also I wanted to have memory available and used, because of that I needed another parameter .. there is only one parameter that is not nedded - '-w', but it won't breaking things either.